### ISSUE TITLE
Specifying multiple zypper options per variable is broken

### ISSUE THREAD

**Issue by smithfarm** [SUSE/teuthology#110]
Specifying multiple options per variable in this line:

https://github.com/SUSE/teuthology/blob/e0d7cc22/teuthology/task/install/rpm.py#L29

leads to:

```
2017-10-26T09:38:40.428 INFO:teuthology.orchestra.run.target137074027035:Running: "for d in ceph-radosgw ceph-test ceph ceph-mgr ceph-fuse libcephfs2 libcephfs-devel librados2 librbd1 python-ceph rbd-fuse ; do sudo zypper '-n --no-gpg-checks' remove --capability $d || true ; done"
2017-10-26T09:38:40.491 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.492 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.493 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.515 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.515 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.515 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.536 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.536 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.536 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.559 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.560 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.560 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.584 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.584 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.585 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.606 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.606 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.606 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.628 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.629 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.629 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.649 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.649 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.650 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.670 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.671 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.671 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.690 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.690 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.690 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.711 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option ' '
2017-10-26T09:38:40.711 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
2017-10-26T09:38:40.711 INFO:teuthology.orchestra.run.target137074027035.stderr:Unknown option '-'
```

**Comment by kshtsk** [comment]
I think this will be resolved with #107

**Comment by smithfarm** [comment]
Thanks, closing.
